ipinfo/cmd_n2ip6.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"github.com/ipinfo/cli/lib"
+	"github.com/ipinfo/cli/lib/complete"
+	"github.com/ipinfo/cli/lib/complete/predict"
+	"github.com/spf13/pflag"
+)
+
+var completionsN2IP6 = &complete.Command{
+	Flags: map[string]complete.Predictor{
+		"--nocolor": predict.Nothing,
+		"-h":        predict.Nothing,
+		"--help":    predict.Nothing,
+	},
+}
+
+// printHelpN2IP6 prints the help message for the "n2ip6" command.
+func printHelpN2IP6() {
+	fmt.Printf(
+		`Usage: %s n2ip6 [<opts>] <expr>
+
+Example:
+  %[1]s n2ip6 "190.87.89.1"
+  %[1]s n2ip6 "2001:0db8:85a3:0000:0000:8a2e:0370:7334
+  %[1]s n2ip6 "2001:0db8:85a3::8a2e:0370:7334
+  %[1]s n2ip6 "::7334
+  %[1]s n2ip6 "7334::""
```

Comment:
There are some mistakes in the examples above. Also they don't look like numbers.